model/task_lifecycle.go

```diff
@@ -639,6 +639,14 @@ func doBisectStepback(ctx context.Context, t *task.Task) error {
 		return errors.Wrapf(err, "setting stepback info for task '%s'", nextTask.Id)
 	}
 
+	grip.Debug(message.Fields{
+		"message":                       "bisect stepback",
+		"last_failing_stepback_task_id": s.LastFailingStepbackTaskId,
+		"last_passing_stepback_task_id": s.LastPassingStepbackTaskId,
+		"next_task_id":                  nextTask.Id,
```

Comment:
I dunno what this log is for exactly, but would it be helpful to also include the current task that's being stepped back (like current task ID /display name/build ID)?

Reply:
The log is to be able to trace tasks in stepback and the path they take / should take. I think adding some of those fields and the status of the task that's having stepback performed might be useful actually, let me add those.